tests/integration/common/transaction_utils.rs

```diff
@@ -83,6 +84,20 @@ fn tx_hash_invoke_v0(
     ])
 }
 
+/// Produce a hash for an Invoke V1 TXN with the provided elements
+fn tx_hash_invoke_v1(contract_address: Felt252, calldata: Vec<Felt252>, max_fee: Felt252, nonce: Felt252) -> Felt252 {
+    hash_on_elements(vec![
+        Felt252::from_bytes_be_slice(INVOKE_PREFIX),
+        Felt252::ONE,
+        contract_address,
+        Felt252::ZERO,
+        hash_on_elements(calldata),
+        Felt252::from(max_fee),
+        Felt252::from(u128::from_str_radix(SN_GOERLI, 16).unwrap()),
```

Comment:
Unrelated to your PR, I never realized we don't pass the chain ID here. That's a potential problem, gotta fix that in another PR.

Reply:
I'll take care of that, it's in the critical path for the integration with Pathfinder.